### PR TITLE
docs: add FAQ entry for seeing resource name in access requests

### PR DESCRIPTION
**Pull request by zmb3**
Additionally, update the structure of this FAQ page to match the other FAQ pages we already have in the docs.

### CODE REVIEW

File: docs/pages/access-controls/faq.mdx [hunk 1]
```diff
@@ -5,10 +5,23 @@ description: Frequently asked questions about Teleport RBAC
...
+
+## Why do I see a UUID instead of a hostname when reviewing access requests?
+
+Resource Access Requests embed the UUID of requested resources in order to ensure
+that extra access isn't mistakenly granted due to overlapping hostnames.
+
+In order for Access Request reviewers to see the hostname, they must either:
+
+- have permissions to access the requested server themselves, or
+- have a `preview_as_roles` set with a role that can access the server
```
Comment (codingllama) [inline docs/pages/access-controls/faq.mdx:27]: Suggestion:

```suggestion
- Have permissions to access the requested server themselves, or
- Have `preview_as_roles` set with a role that can access the server
```

Not sure about the extra "a", sounds strange to me (but I may be mistaken).

**Review by codingllama — APPROVED** [review]
Thanks!

**Review by kimlisa — APPROVED** [review]

**Review by nklaassen — APPROVED** [review]
looks helpful, thanks. I'm pretty sure this is released in 11.0.0

**Review by dgkirkwood — APPROVED** [review]
Looks good Zac!

File: docs/pages/access-controls/faq.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/access-controls/faq.mdx:27]: The 👍 indicates that you agree, but this is resolved without change.

[the remark this answers is not in the document]
Comment (alexfornuto) [reply docs/pages/access-controls/faq.mdx:27]: The 👍 indicates that you agree, but this is resolved without change.

**Review by alexfornuto — COMMENTED** [review]
I have the same concerns as raised in the comments resolved without change.

**Comment by zmb3** [comment]
@alexfornuto oops! I forgot to push after addressing the comments 🤦‍♂️

**Review by r0mant — APPROVED** [review]

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**